tests/tck-build-logic/src/main/java/org/graalvm/internal/tck/DockerUtils.java

```diff
@@ -7,22 +7,25 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.Arrays;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-import java.util.stream.Collectors;
 
 public class DockerUtils {
+    private static String getDockerfileDirectory() {
+        return Paths.get("./tests/tck-build-logic/src/main/resources/allowed-docker-images").toString();
```

Comment:
Will this always resolve the correct directory? Should we make it relative to `DockerUtils.java`?

Reply:
I changed whole handling to use resources instead of Paths and Files